packages/e2e-test-utils/src/enable-experimental-features.js

```diff
@@ -8,12 +8,7 @@ import { addQueryArgs } from '@wordpress/url';
  */
 import { visitAdminPage } from './visit-admin-page';
 
-/**
- * Enables experimental features from the plugin settings section.
- *
- * @param {Array} features Array of {string} selectors of settings to enable. Assumes they can be enabled with one click.
- */
-export async function enableExperimentalFeatures( features ) {
+export async function setExperimentalFeaturesState( features, enable ) {
```

Comment:
This one could be moved to the shared folder to its own file, although I don't think we are strict about it so maybe it's fine as is.